scripts/build_utils.sh

```diff
@@ -729,6 +729,18 @@ write_collect_logs_playbook() {
 - hosts: all
   become: yes
   tasks:
+    - name: get yum repoinfo result
```

Comment:
these `yum` commands are now failing on every non-yum test. Although it is getting ignored because of the `failed_when: false`, it is distracting to see these run, fail, and get ignored on Debian-based tests

Reply:
@alfredodeza this is not supposed to stay, but we could add a when so these are run only on CentOS?

Reply:
Yeah, sure, that would help!